prime-angular-frontend/src/app/modules/site-registration/pages/technical-support-page/technical-support-page.component.ts

```diff
@@ -55,6 +55,8 @@ export class TechnicalSupportPageComponent extends AbstractEnrolmentPage impleme
       contact.physicalAddress = new Address();
     }
     this.formState.form.patchValue(contact);
+    // mark the form touched so address form can expand
+    this.formState.form.markAllAsTouched();
```

Comment:
> Feels like we could pass in a binding to `ContactProfileForm` that sets `showAddress` for the physical address on the profile form for each of these contact pages.

This is related to the above form.untouched fix. After patching and markAsTouched, the hidden address form will show otherwise it remains hidden.

Reply:
I realize the relationship it was the reason I pointed this out since it emphasizes my initial point where you need to know that the you have to mark the state as touched in order to affect a component 2 levels below the current component instead of using bindings.  I'm trying to illustrate that even though it works it isn't necessarily intuitive or a good idea for a base component, which could also have application wide side-effects.